app/assets/stylesheets/artists.scss

```diff
@@ -1,3 +1,82 @@
 // Place all the styles related to the artists controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+.row-bio {
+    height: 360px;
+    display: flex;
+    .col-info {
+        display: flex;
+        flex-direction: column;
+        min-width: 248px;
```

Comment:
Declaring all the px values like this seems fine, but I wonder if it may be better to extend tachyons classes (ex. `extend .mw3;` (extend the max-width: 3 class)). My concern is that it displays normally on a standard laptop display, but for more unique setups (vertical monitors, very large monitors, very small monitors / phones) we aren't taking advantage of the versatile grid setup tachyons offers. Let me know what you think!